moml/simulation/molecular_dynamics/force_field/plugins/ro_v1/build.py

```
@@ -0,0 +1,10 @@
+from importlib import import_module
+from pathlib import Path
+from typing import Tuple, List
+# Re-use NF builder then scale box / adjust density
+_nf_builder = import_module("moml.simulation.molecular_dynamics.force_field.plugins.nf_pa_v1.build")
+
+def build(tmp_dir: Path, cfg: dict) -> Tuple[Path, "Topology", List[int]]:
```

Ruff (0.11.9)

Undefined name `Topology`

(F821)